### PR TITLE
blog article for tivity health use case

### PR DESCRIPTION
**Pull request by SaraDPH**
Post-PulumiUP article

The talk is being re-used to PulumiUP Europe, so it will be delayed for afterward.

ETA 11/21

### CODE REVIEW

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+title: "Accelerating Security Automation for Faster, Cheaper, and Better"
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:2]:
```suggestion
title: "Accelerating Security Automation for Faster, Cheaper, and Better Results"
```

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+---
+title: "Accelerating Security Automation for Faster, Cheaper, and Better Results"
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:2]:
```suggestion
title: "Security Automation for Faster, Cheaper, and Better Results"
```

**Comment by SaraDPH** [comment]
I'm opened to suggestions on what to name the blog post... I have been considering a few options, and I am not yet happy with it.

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
+---
+title: "Security Automation for Faster, Cheaper, and Better Results"
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:2]:
```suggestion
title: "DevSecOps Game-Changer: Security Automation That Delivers Real Business Results"
```

**Review by Marlowe992 — APPROVED** [review]

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+Tivity Health, a leading health and fitness solutions provider, has embraced this DevSecOps approach using Pulumi, a modern infrastructure as code (IaC) platform. During PulumiUP 2024, David Giambruno, Tivity Health's VP of Engineering and DevOps, shared how, by leveraging Pulumi, he led the transformation that continuously drives remarkable results in speed, cost savings, and security.
+
+<!--more-->
+
+## On this article:
+
+- [The Beginning: From Data Center to the Cloud](/blog/devsecops-strategy-security-automation-tivity-health/#the-beginning-from-data-center-to-the-cloud)
+- [Embracing Pulumi: The Power of Automation, Productivity, and Security](/blog/devsecops-strategy-security-automation-tivity-health/#embracing-pulumi-the-power-of-automation-productivity-and-security)
+- [Driving Business Impact Through Security Automation](/blog/devsecops-strategy-security-automation-tivity-health/#driving-business-impact-through-security-automation)
+- [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/fostering-devops-culture-through-cross--functional-collaboration)
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:78]:
```suggestion
- [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/fostering-devops-culture-through-cross-functional-collaboration)
```

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+Tivity Health, a leading health and fitness solutions provider, has embraced this DevSecOps approach using Pulumi, a modern infrastructure as code (IaC) platform. During PulumiUP 2024, David Giambruno, Tivity Health's VP of Engineering and DevOps, shared how, by leveraging Pulumi, he led the transformation that continuously drives remarkable results in speed, cost savings, and security.
+
+<!--more-->
+
+## On this article:
+
+- [The Beginning: From Data Center to the Cloud](/blog/devsecops-strategy-security-automation-tivity-health/#the-beginning-from-data-center-to-the-cloud)
+- [Embracing Pulumi: The Power of Automation, Productivity, and Security](/blog/devsecops-strategy-security-automation-tivity-health/#embracing-pulumi-the-power-of-automation-productivity-and-security)
+- [Driving Business Impact Through Security Automation](/blog/devsecops-strategy-security-automation-tivity-health/#driving-business-impact-through-security-automation)
+- [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/fostering-devops-culture-through-cross-functional-collaboration)
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:78]:
```suggestion
- [Fostering DevOps Culture Through Cross-Functional Collaboration](/blog/devsecops-strategy-security-automation-tivity-health/#fostering-devops-culture-through-cross-functional-collaboration)
```

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+title: "DevSecOps Game-Changer: Security Automation That Delivers Real Business Results"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date and time this file was generated. Dates are used for display and
+# ordering purposes only; they have no effect on whether or when a post is
+# published. To influence the ordering of posts published on the same date, use
+# the time portion of the date value; posts are sorted in descending order by
+# date/time.
+date: 2024-10-10T07:16:30Z
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:11]:
```suggestion
date: 2024-11-21T07:16:30Z
```

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
+---
+title: "DevSecOps Game-Changer: Security Automation That Delivers Real Business Results"
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:2]:
```suggestion
title: "DevSecOps Game-Changer: Security Automation That Delivers Business Results"
```

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: Accelerate security automation to achieve faster, cheaper, and more secure cloud deployments. Discover Tivity Health's cloud transformation journey with Pulumi
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:26]:
```suggestion
meta_image: david-tivity-healthy-quote.png
```

File: content/blog/devsecops-strategy-security-automation-tivity-health/index.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+
+social:
+    twitter:
+    linkedin:
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+In today's cloud-driven business landscape, organizations are pressured to deliver new products and features faster than ever. However, they must also balance rapid deployment, security, and cost-effective infrastructure management.
```
Comment (SaraDPH) [inline content/blog/devsecops-strategy-security-automation-tivity-health/index.md:65]:
```suggestion
Organizations are under constant pressure to deliver new products and features faster than ever. But speed alone isn’t enough—businesses must also navigate the complex challenges of ensuring security and managing infrastructure costs effectively.
```

**Pull request merged**